docs/operations/blocks-storage.md

```diff
@@ -262,6 +262,70 @@ tsdb:
       # CLI flag: -experimental.tsdb.bucket-store.index-cache.postings-compression-enabled
       [postings_compression_enabled: <boolean> | default = false]
 
+    caching_bucket:
```

Comment:
In `docs/operations/blocks-storage.template` we do have a paragraph explaining the index cache. Would be great add a similar one for the chunks caching.